src/sql/parts/jobManagement/views/jobHistory.component.ts

```diff
@@ -111,22 +140,26 @@ export class JobHistoryComponent extends Disposable implements OnInit, OnDestroy
 		}
 	}
 
-	private jobAction(action: string): void {
+	private jobAction(action: string, jobName): void {
```

Comment:
can you add a type to jobName?  In general I suggest we try to explicitly provide types when declaring variables.  But we should certainly do this for function parameters.